lint/package.go

```diff
@@ -182,17 +184,22 @@ func (p *Package) scanSortable() {
 	}
 }
 
-func (p *Package) lint(rules []Rule, config Config, failures chan Failure) {
+func (p *Package) lint(rules []Rule, config Config, failures chan Failure) error {
 	p.scanSortable()
 	var wg sync.WaitGroup
 	for _, file := range p.files {
 		wg.Add(1)
 		go (func(file *File) {
-			file.lint(rules, config, failures)
+			err := file.lint(rules, config, failures)
+			if err != nil {
+				fmt.Fprintln(os.Stderr, err)
```

Comment:
Maybe prepend with some explanation string:

```go
fmt.Fprintln(os.Stderr, "Error during linting:", err)
```

The error itself may not be that readable if dropped out without it.